src/hyp_layout.rs

```diff
@@ -21,7 +21,18 @@
 // | (unused 4Mb)            |
 // +-------------------------+ UMODE_INPUT_START
 // | Umode Input Area        |
-// +-------------------------+ +UMODE_INPUT_SIZE
+// +-------------------------+ +UMODE_INPUT_SIZE (UMODE_INPUT_END, UMODE_END)
+// | (unused 4Mb)            |
+// +-------------------------+ HYP_STACKBTM_MIN
+// | (unused)                |
+// +-------------------------+ HYP_STACKTOP - stack size [1]
+// | Hypervisor Stack        |
+// +-------------------------+ HYP_STACK_TOP (0xffff_ffff_ffff_f000)
```

Comment:
nit: inconsistency between `STACKTOP` and `STACK_TOP`. i'd prefer the latter.

Reply:
Whops.

Reply:
Fixed.